### DISCUSSION TITLE
ValueError: Excel does not support datetimes with timezones.

### DISCUSSION THREAD

**Discussion by McHouse** [theOehrly/Fast-F1#380] [Q&A] [answered]
why this script

print("DataFrame SCHEDULE")
print("\n",schedule,"\n")
print(schedule.keys())
DSchedule = pd.DataFrame(schedule)
print(DSchedule.dtypes)
NFile_1=str(YEAR)+"-SCHEDULE_SEASON.xlsx"
filepath = Path_DirGP+NFile_1
DSchedule.to_excel(filepath, index=True)
# DSchedule.to_excel(filepath, index=True, engine='openpyxl')
end = time.time()
delta = end - start0
print("Processing time %.2f s." % delta)

that has worked perfectly for months, now generates this error??
ValueError: Excel does not support datetimes with timezones. Please ensure that date times are time zone unaware before writing to Excel.
You can see that I have also tried with openpyxl, but with the same result.
Because??
Any solution??

Thank you

**Comment by theOehrly** [comment, marked as answer]
From the release notes of v3.0.0: 

> In the event schedule, Session\*Date will now be a timezone-aware local timestamp (previously: local, but not timezone-aware). Additionally, for each session there now exists a non-timezone aware UTC timestamp in Session\*DateUtc (#253). Timezone information and UTC timestamps are only available when using one of 'fastf1' or 'f1timing' as backend.

Also have a look at https://docs.fastf1.dev/events.html#event-schedule-data

Basically, you have two options. 

1. Strip the convert the local timezone columns to timezone-naive values
2. Only use the UTC columns

**Reply by McHouse** [reply]
Thank you so much.
I see the 3.x.x versions have many improvements and fixes.
I'll keep 2.3.3 working, while I learn to use what's new in 3.0.3.
I repeat, thank you very much for the support and the speed of response.
